Streetcode/Streetcode.XIntegrationTest/appsettings.Local.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=DESKTOP-I7Q35NQ\\SQLEXPRESS;Database=_TestStreetcodeDb;User Id=sa;Password=Admin@1234;MultipleActiveResultSets=true"
```

Comment:
Do you need to commit this?

Reply:
We need this string, but the one that is used to connect to local docker db